packages/docprovider/src/ydrive.ts

```diff
@@ -123,7 +126,7 @@ export class RtcContentProvider
       const provider = this._providers.get(key);
 
       if (provider) {
-        // Save is done from the backend
+        provider.wsProvider?.ws?.send('save_to_disc');
```

Comment:
I think that we should use a custom message type (see [here](https://github.com/yjs/y-protocols/blob/master/PROTOCOL.md#combining-protocols)). Maybe `2` followed by `save`?
Also, should we wait for a reply indicating that the file has indeed been saved? Otherwise the following `get` will probably not return the state of the saved file.